docs/annotations.md

```diff
@@ -89,3 +91,21 @@ metadata:
 spec:
   ...
 ```
+
+## TLS Termination
+
+### Setting Minimum TLS Version
+
+The `k8s.ngrok.com/tls-min-version` annotation can be used to set the minimum TLS version for all routes belonging to an Edge. *Note*: **This setting applies to all routes
+belonging to an Edge, not just the routes defined in the Ingress resource**. The annotation accepts a string value. If not specified(default), the `TLS Termination` module will be disabled in ngrok and TLS will be terminated at the edge using the ngrok default minimum TLS version.
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: minimal-ingress
+  annotations:
+    k8s.ngrok.com/tls-min-version: "1.3"
```

Comment:
for the names of these annotations, I assume we want to work with russ in our next product review to make sure these are consistent across products. Thoughts on leveraging what we use in the ngrok cli for these? 

```
ngrok api edge-modules
NAME:
  edge-modules -

USAGE:
  ngrok api edge-modules [flags]

DESCRIPTION:

COMMANDS:
  https-edge-mutual-tls
  https-edge-route-backend
  https-edge-route-circuit-breaker
  https-edge-route-compression
  https-edge-route-ip-restriction
  https-edge-route-oauth
  https-edge-route-oidc
  https-edge-route-request-headers
  https-edge-route-response-headers
  https-edge-route-saml
  https-edge-route-webhook-verification
  https-edge-route-websocket-tcp-converter
  https-edge-tls-termination
  tcp-edge-backend
  tcp-edge-ip-restriction
  tls-edge-backend
  tls-edge-ip-restriction
  tls-edge-mutual-tls
  tls-edge-tls-termination
```

so this would be `k8s.ngrok.com/https-edge-tls-termination-min-version`

Reply:
I'm not opposed, but would like Russ' feedback on it. Given the different section, I'd almost be more inclined to have `.` delimiting some of these: `https-edge.tls-termination.min-version` or something like that. They should be very easy to change and map though unless we change to/from embedded json on a lot of them.

Reply:
I do like the idea of having a delimiter between the module type vs the actual configuration item. I haven't seen it used much but its not dissallowed. Accross a couple of controllers i see a few cases where people used `.` to delimit a few random words like `alb.ingress.kubernetes.io/group.name` and then some use it to reprsent a variable object like a key in a dictionary `konghq.com/headers.*`

:+1: to getting feedback from russ and they are easy to change so we can do them later